src/Symfony/Component/HttpClient/Response/ResponseTrait.php

```diff
@@ -244,6 +244,14 @@ private static function addResponseHeaders(array $responseHeaders, array &$info,
         }
     }
 
+    private static function handleContentType(&$content, string $contentType, string $bufferRegex): void
```

Comment:
let's not pass $content at all and name the method createBufferForContentType?
this means checking for null === $content outside of the method of course

Reply:
You mean that I should return the stream created by `fopen` in the createBufferForContentType or otherwise `null` if regex matches?

Reply:
Correct

Reply:
Since there's no return type for a stream (yet?), can/should I add a phpdoc block to declare the return type?